src/CSET/recipes/surface_spatial_difference_case_aggregation_mean_hour_of_day.yaml

```
@@ -1,5 +1,5 @@
 category: Surface Spatial Plot Difference
-title: "$VARNAME difference ($BASE_MODEL - $OTHER_MODEL)\n Aggregation by hour of day"
+title: "$VARNAMEz\n Difference ($BASE_MODEL - $OTHER_MODEL)\n Aggregation by hour of day"
```

```suggestion
title: "$VARNAME\nDifference ($BASE_MODEL - $OTHER_MODEL)\nAggregation by hour of day"
```
Rouge `z` also snuck in here.